bool resultNav
---
int64 location

